### ISSUE TITLE
Java 8 with tomcat 7 Support

### ISSUE THREAD

**Issue by Sixuan** [jcoleman/tomcat-redis-session-manager#76]
Hi,
This library looks pretty solid for Java 6/7 with tomcat 7. I'm assuming it would also be compatible for "Java 8 with tomcat 7"? Is there anything else I should do or it should just work as it? 
Any feedback is really really appreciated. 

Thanks

**Comment by tpietsch** [comment]
https://github.com/tpietsch/tomcat-redis-session-manager

**Comment by vijaygoturi** [comment]
@Sixuan  Are you able to work with Java 8 version?

**Comment by Sixuan** [comment]
Yes. Been using github.com/tpietsch/tomcat-redis-session-manager for a year on Java 8 and tomcat 7, haven't seen problems so far.

**Comment by vijaygoturi** [comment]
It is getting redirected to j_security_check after the login

**Comment by robertofd** [comment]
hi , is posible use redis-session-manager with java 8 and tomcat 8?
